src/GraphQL/Introspection/__EnumValue.cs

```diff
@@ -11,6 +11,11 @@ public class __EnumValue : ObjectGraphType<EnumValueDefinition>
         /// Initializes a new instance of the <c>__EnumValue</c> introspection type.
         /// </summary>
         public __EnumValue()
+            : this(false)
+        {
+        }
+
+        internal __EnumValue(bool allowAppliedDirectives)
```

Comment:
Basically, I think this new feature should be part of the main codebase, not some "secret switch" that if we enable it works only one way.  So if you want all these types to have this boolean, then add it.  Perhaps `public __EnumValue(bool allowAppliedDirectives = false)`.  But don't have all this stuff internal so only when you use a certain filter it enables a whole bunch of code that is otherwise inaccessible.

Reply:
Good. It's even easier for me.

Reply:
Done